Makefile

```diff
@@ -113,13 +113,13 @@ $(NATIVE_LUAJIT_LIB): $(BIN_LUAJIT_DIR)
 	$(MAKE) -C $(BIN_LUAJIT_DIR) BUILDMODE=static
 
 fennel.exe: src/launcher.fnl fennel $(LUA_INCLUDE_DIR)/liblua-mingw.a
-	$(COMPILE_ARGS) CC=i686-w64-mingw32-gcc ./fennel --no-compiler-sandbox \
+	$(COMPILE_ARGS) CC=gcc ./fennel --no-compiler-sandbox \
```

Comment:
Aha, ok. So this is set up as it currently is in order to support cross-compilation to build `fennel.exe` from Linux systems as the primary use case.

This change would make it so I can't publish Windows builds when I make a release. I think in order to do this in a way that works for both cases, we'd have to create a separate variable, like maybe `WINDOWS_CC` which could be overridden separately; like maybe it could default to `gcc` or `$(CC)` on Windows but use its current `i686-w64-mingw32-gcc` value on Linux?

The `fennel-bin` target is intended as the "native" one for the platform in question without cross-compilation. Does that work on Windows?